tutorials/krusty/KRUSTY/job_sawtooth

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
We don't need to version control this file, as we frequently change the Sawtooth instructions when modules change. Let's please delete it

Reply:
Okay, I've just deleted it!